From 3c3db9d1b8e24374d6e24610f8d04a2dba8fa151 Mon Sep 17 00:00:00 2001
From: Frode Isaksen <frodex.isaksen@intel.com>
Date: Thu, 21 Aug 2014 17:17:28 +0200
Subject: [PATCH 1/3] ACPI: add BCM43241 in ACPI platform device list

Issue: GMINL-458

Add "BCM43241" as an ACPI platform device, in order to
enable Broadcom WiFi chips.

Change-Id: I1724110f9a63a0cf6cba5e35a12c97f4afc46b0e
Signed-off-by: Frode Isaksen <frodex.isaksen@intel.com>
---
 drivers/acpi/acpi_platform.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/acpi/acpi_platform.c b/drivers/acpi/acpi_platform.c
index 49109365928b..205f643a07e8 100644
--- a/drivers/acpi/acpi_platform.c
+++ b/drivers/acpi/acpi_platform.c
@@ -39,6 +39,7 @@ static const struct acpi_device_id acpi_platform_device_ids[] = {
 	{ "RTL8723" },
 	{ "OBDA8723" },
 	{ "BCM4752E" },
+	{ "BCM43241" },
 
 	/* Intel Smart Sound Technology */
 	{ "INT33C8" },
-- 
1.7.9.5

